src/base-element.js

```diff
@@ -678,8 +678,9 @@ export class BaseElement {
     attributes = isArray(attributes) ? attributes : [attributes];
     for (let i = 0; i < attributes.length; i++) {
       const attr = attributes[i];
-      if (this.element.hasAttribute(attr)) {
-        element.setAttribute(attr, this.element.getAttribute(attr));
+      const val = this.element.getAttribute(attr);
```

Comment:
I'm assuming the rest of the changes are only meant be in https://github.com/ampproject/amphtml/pull/23767.

Reply:
Yeah.